examples/morpho_blue.ipynb

```diff
@@ -0,0 +1,1018 @@
+    "    \"topic0\": [CREATION_MARKET_TOPIC],\n",
+    "}\n",
+    "\n",
+    "# Collect the raw events with cryo\n",
+    "blue_market_creations_raw = cryo.collect(\n",
+    "    **params\n",
+    ")\n",
+    "\n",
+    "blue_market_creations_raw.head(5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Parsing Event name: event CreateMarket\n",
+      "Indexed types: [('bytes32', 'id')]\n",
+      "Data types: [('address', 'loanToken'), ('address', 'collateralToken'), ('address', 'oracle'), ('address', 'irm'), ('uint256', 'lltv')]\n"
+     ]
+    },
+    {
+     "data": {
+      "text/html": [
+       "<div><style>\n",
+       ".dataframe > thead > tr,\n",
+       ".dataframe > tbody > tr {\n",
+       "  text-align: right;\n",
+       "  white-space: pre-wrap;\n",
+       "}\n",
+       "</style>\n",
+       "<small>shape: (5, 9)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>id</th><th>block_number</th><th>transaction_index</th><th>log_index</th><th>loanToken</th><th>collateralToken</th><th>oracle</th><th>irm</th><th>lltv</th></tr><tr><td>str</td><td>u32</td><td>u32</td><td>u32</td><td>str</td><td>str</td><td>str</td><td>str</td><td>f64</td></tr></thead><tbody><tr><td>&quot;0xc54d7acf14de…</td><td>18919623</td><td>73</td><td>135</td><td>&quot;0xc02aaa39b223…</td><td>&quot;0x7f39c581f595…</td><td>&quot;0x2a01eb949609…</td><td>&quot;0x870ac11d48b1…</td><td>9.4500e17</td></tr><tr><td>&quot;0x54efdee08e27…</td><td>18919858</td><td>95</td><td>141</td><td>&quot;0xa0b86991c621…</td><td>&quot;0x000000000000…</td><td>&quot;0x000000000000…</td><td>&quot;0x000000000000…</td><td>0.0</td></tr><tr><td>&quot;0x58e212060645…</td><td>18919860</td><td>80</td><td>255</td><td>&quot;0xc02aaa39b223…</td><td>&quot;0x000000000000…</td><td>&quot;0x000000000000…</td><td>&quot;0x000000000000…</td><td>0.0</td></tr><tr><td>&quot;0x08535810fead…</td><td>18925864</td><td>126</td><td>392</td><td>&quot;0xa0b86991c621…</td><td>&quot;0xabc56186c2df…</td><td>&quot;0x6e8f5b2df218…</td><td>&quot;0x870ac11d48b1…</td><td>9.6500e17</td></tr><tr><td>&quot;0xb323495f7e41…</td><td>18925910</td><td>94</td><td>214</td><td>&quot;0xa0b86991c621…</td><td>&quot;0x7f39c581f595…</td><td>&quot;0x48f7e36eb6b8…</td><td>&quot;0x870ac11d48b1…</td><td>8.6000e17</td></tr></tbody></table></div>"
+      ],
+      "text/plain": [
+       "shape: (5, 9)\n",
+       "┌───────────┬───────────┬───────────┬───────────┬───┬───────────┬───────────┬───────────┬──────────┐\n",
+       "│ id        ┆ block_num ┆ transacti ┆ log_index ┆ … ┆ collatera ┆ oracle    ┆ irm       ┆ lltv     │\n",
+       "│ ---       ┆ ber       ┆ on_index  ┆ ---       ┆   ┆ lToken    ┆ ---       ┆ ---       ┆ ---      │\n",
+       "│ str       ┆ ---       ┆ ---       ┆ u32       ┆   ┆ ---       ┆ str       ┆ str       ┆ f64      │\n",
```

Comment:
why collateral etc are str and not addresses

Reply:
because polars does not handle eth types yet (even though there are [bounties](https://github.com/pola-rs/polars/issues/15443) for that)